src/GraphQL.Tests/GraphQL.Tests.csproj

```diff
@@ -21,6 +21,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\GraphQL.Dummy\GraphQL.Dummy.csproj" />
+    <ProjectReference Include="..\GraphQL.MicrosoftDI\GraphQL.MicrosoftDI.csproj" />
```

Comment:
Let's move tests for GraphQL.MicrosoftDI into a new test project from GraphQL.Tests. We already have one for dataloaders.